runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/artifact/BeamFileSystemArtifactStagingService.java

```diff
@@ -305,7 +311,7 @@ public void onCompleted() {
    * Serializable StagingSessionToken used to stage files with {@link
    * BeamFileSystemArtifactStagingService}.
    */
-  private static class StagingSessionToken implements Serializable {
+  static class StagingSessionToken implements Serializable {
```

Comment:
We should keep the StagingSessionToken as no one else should care about its structure outside this class.